frontend/src/Components/Products/Products.module.scss

```diff
@@ -0,0 +1,25 @@
+@import "~plaid-threads/scss/variables";
+
+.productsContainer {
+  width: 100%;
+  border: 1px solid $black200;
+  border-top: 0px;
+  border-bottom: 0px;
+  display: flex;
+  flex-direction: column;
+  margin: 2 * $unit 0 0 0;
+  align-items: center;
+  box-shadow: $shadow-small;
+}
+
+.productsHeader {
+  width: 100%;
+  border: 1px solid $black200;
+  border-bottom: 0px;
+  height: 13 * $unit;
+  font-weight: 800;
+  padding-top: 5 * $unit;
+  padding-left: 5 * $unit;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
this looks the same as the items one. Should the container be extracted to a shared component so the styles are not copy pasted?

Reply:
done